bikesharing/admin.py

```diff
@@ -31,21 +31,37 @@ class BikeAdmin(LeafletGeoAdmin, admin.ModelAdmin):
 
     @mark_safe
     def location(self, obj):
-        if obj is None or obj.current_position() is None:
```

Comment:
this looks (maybe only in the diff) like a lot of copied code there. not sure if we should put this already in an function and call it twice with `obj.public_geolocation()` and `obj.internal_geolocation()`

Reply:
the truth is... that it is copied

Reply:
hope thats better: 76bc7e673d0fd878a59f41a3cc3f96a39a728c50